src/material/expansion/expansion-panel.ts

```diff
@@ -209,7 +209,7 @@ export class MatExpansionPanel extends CdkAccordionItem implements AfterContentI
     if (this._lazyContent) {
       // Render the content as soon as the panel becomes open.
       this.opened.pipe(
-        startWith(null!),
```

Comment:
FYI I think this was necessary to avoid some rxjs typing breakage in google at some point. We'll see if it breaks anything in the presubmit.